### ISSUE TITLE
OpenBSD: icewm -i list doesn't work by default because of tar implementation

### ISSUE THREAD

**Issue by exceptbees** [ice-wm/icewm#179]
OpenBSD tar(1) strips off the trailing slash from directory names when listing archive contents, see more on that [in the mailing list](https://marc.info/?l=openbsd-tech&m=111075158616719&w=2) and [the system's tar.c](https://cvsweb.openbsd.org/cgi-bin/cvsweb/src/bin/pax/tar.c?rev=1.85&content-type=text/x-cvsweb-markup).

Compare:
```
$ lzip -cd icewm-extra.tar.lz | tar -tf - | head -1 
icewm-extra/18k
```
vs.
```
$ lzip -cd icewm-extra.tar.lz | gtar -tf - | head -1
icewm-extra/18k/
```

So `icewm -i list` only shows the list of themes if `gtar`is installed and exits silently if it's not. Would be nice to have it work with the platform's default `tar`. 

P.S. Tested the archive with NetBSD's tar(1), and it does display trailing slashes in directory names. Don't have a FreeBSD system, so can't check.

**Comment by gijsbers** [comment]
Yes. The function `get_tar()` in `icewm/src/theminst.cc` first looks for `gtar` on BSDs. Wasn't that installed on your OpenBSD? If it was, it should have worked, no?
